binarytree/__init__.py

```diff
@@ -73,6 +96,126 @@ def _add_left(parent, child):
 def _add_right(parent, child):
     """Add the child to the right of the parent."""
     setattr(parent, _right_attr, child)
+    
+def pruneLeft(node):
+    """Prunes the left subtree of a node."""
+    node.__setattr__(_left_attr, _null)
+        
+def pruneRight(node):
+    """Prunes the right subtree of a node."""
+    node.__setattr__(_right_attr, _null)
+
+def getParentNode(root,nodeId):
```

Comment:
Please change from camelcase to snakecase for consistency. Missing a whitespace after comma.